packages/slice-machine/src/modules/useSliceMachineActions.ts

```diff
@@ -196,12 +168,34 @@ const useSliceMachineActions = () => {
   const cleanupCustomTypeStore = () =>
     dispatch(cleanupCustomTypeStoreCreator());
   const saveCustomType = () => dispatch(saveCustomTypeCreator.request());
+
+  /** Success actions = sync store state from external actions.
```

Comment:
I was wondering, should we create another file and another hook like useSliceMachineSuccessActions, or else? It could help us to know the good redux stuff and the "bad" 🤔

Reply:
Ok for later if needed